content/zh/docs/tasks/kubelet-credential-provider/kubelet-credential-provider.md

```diff
@@ -0,0 +1,201 @@
+---
+title: 配置 Kubelet 镜像凭证提供程序
+description: 配置 Kubelet 镜像凭证提供程序插件
+content_type: task
+---
+
+{{< feature-state for_k8s_version="v1.20" state="alpha" >}}
+
+<!-- overview -->
+<!--
+Starting from Kubernetes v1.20, the kubelet can dynamically retrieve credentials for a container image registry
+using exec plugins. The kubelet and the exec plugin communicate through stdio (stdin, stdout, and stderr) using
+Kubernetes versioned APIs. These plugins allow the kubelet to request credentials for a container registry dynamically
+as opposed to storing static credentials on disk. For example, the plugin may talk to a local metadata server to retrieve
+short-lived credentials for an image that is being pulled by the kubelet.
+-->
+从 Kubernetes v1.20 开始，kubelet 可以使用 exec 插件动态回收容器镜像注册表的凭证。kubelet 和 exec 插件通过
+使用 Kubernetes 版本 API 通过 stdio（stdin，stdout 和 stderr）来进行通信，这些插件允许 kubelet 动态地请求
+容器注册表的凭证，而不是在磁盘上存储静态的凭证。比如，该插件可以和本地元数据服务器进行通话，以检索由 kubelet 提取镜像的短期凭证。
+
+<!--
+You may be interested in using this capability if any of the below are true:
+
+* API calls to a cloud provider service are required to retrieve authentication information for a registry.
+* Credentials have short expiration times and requesting new credentials frequently is required.
+* Storing registry credentials on disk or in imagePullSecrets is not acceptable.
+
+This guide demonstrates how to configure the kubelet's image credential provider plugin mechanism.
+-->
+如果满足以下任一条件，你可能会对使用此功能感兴趣：
+
+* 需要调用云提供商服务的 API 才能检索注册表的身份验证信息。
+* 凭证的到期时间很短，因此需要经常请求新的凭证。
+* 不允许将注册表凭证存储在磁盘上或 imagePullSecrets 中。
```

Comment:
```suggestion
* 不允许将仓库凭证存储在磁盘上或 imagePullSecrets 中。
```

Reply:
This is not resolved? Registry here means container registry, not Windows Registry.